### ISSUE TITLE
Issue 3.0 for jest

### ISSUE THREAD

**Issue by billnbell** [software-mansion/react-native-reanimated#4197]
### Description

Issue docs wrong.

```
require('react-native-reanimated/lib/reanimated2/jestUtils').setUpTests();
```
Does not exist.

```
require('react-native-reanimated/lib/commonjs/reanimated2/jestUtils').setUpTests();
```

This is there - but I get:

```
 RUNS  src/__tests__/Helpers-test.ts
/Users/bill/theclub/TheClub/node_modules/react-native-reanimated/lib/commonjs/reanimated2/initializers.js:172
        debug: (0, _threads.runOnJS)(capturableConsole.debug),
                                    ^

TypeError: (0 , _threads.runOnJS) is not a function
    at _f (/Users/bill/theclub/TheClub/node_modules/react-native-reanimated/lib/commonjs/reanimated2/initializers.ts:160:25)
    at _f (/Users/bill/theclub/TheClub/node_modules/react-native-reanimated/lib/commonjs/reanimated2/threads.ts:133:5)
    at Timeout._onTimeout (/Users/bill/theclub/TheClub/node_modules/react-native-reanimated/lib/commonjs/reanimated2/initializers.js:155:16)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
error Command failed with exit code 1.
```

### Steps to reproduce

Just install 3,.0

### Snack or a link to a repository

nope

### Reanimated version

3.0

### React Native version

0.71.4

### Platforms

iOS

### JavaScript runtime

None

### Workflow

None

### Architecture

None

### Build type

None

### Device

None

### Acknowledgements

Yes

**Comment by yerevin** [comment]
You can try with fix described in https://github.com/software-mansion/react-native-reanimated/pull/4136, but I'm not sure if it recommended way by creators

**Comment by ajaxjiang96** [comment]
somehow this works:

```ts
// eslint-disable-next-line @typescript-eslint/no-unused-vars
import * as reanimatedJestUtils from 'react-native-reanimated/src/reanimated2/jestUtils';
```

**Comment by croaker** [comment]
I had the same problem after updating to reanimated 3. For me, the problem was that I had previously added react-native-reanimated to "transformIgnorePatterns" in the "jest" section in package.json. Removing it and then replacing `require('react-native-reanimated/lib/reanimated2/jestUtils').setUpTests()` with `require('react-native-reanimated/lib/commonjs/reanimated2/jestUtils').setUpTests()` solved the problem for me.

**Comment by edkimmel** [comment]
Seeing this same issue.  I cannot follow the fix in #4136 because my test suite depends on checking animated styles / running animations.  Mocking the module causes more test failures.

**Comment by edkimmel** [comment]
I found something relevant.

`TypeError: (0 , _threads.runOnJS) is not a function` and etc can be individually solved by patching reanimated to move the export lines to the end of the files.  The variables are not initialized when they are exported as the code is today.

I am trying to find if there was a babel plugin that was meant to be re-ordering these files, but at least it's just an order of operations issue and not a fundamental issue with the functions themselves.

**Comment by kioltk** [comment]
For me the source import helped 
```
require('react-native-reanimated/src/reanimated2/jestUtils').setUpTests();
```

**Comment by tjzel** [comment]
This was fixed by #4844 and #4468.

**Comment by f3rnandorj** [comment]
I've just done what the documentation suggests creating  a file containing this line 

require('react-native-reanimated').setUpTests();

after that, I added the path of this file on jest config on setupFilesAfterEnv.


"setupFilesAfterEnv": [
   "<rootDir>/src/test/jestSetupAfterEnv.ts"   // in my case
 ],
